src/WebJobs.Script.WebHost/applicationHost.xdt

```diff
@@ -16,11 +16,12 @@
       <add name="AspNetCoreModuleV2" image="%XDT_EXTENSIONPATH%\%XDT_BITNESS%\ancm\aspnetcoreV2.dll" xdt:Transform="Insert" />
     </globalModules>
   </system.webServer>
-  <location path="%XDT_SITENAME%" xdt:Locator="Match(path)" xdt:Transform="InsertIfMissing">
-    <system.webServer xdt:Transform="InsertIfMissing">
-      <modules xdt:Transform="InsertIfMissing">
-        <add name="AspNetCoreModuleV2" xdt:Transform="InsertIfMissing"/>
+  <location path="" xdt:Locator="Match(path)">
+    <system.webServer>
+      <modules>
+        <add name="AspNetCoreModuleV2" xdt:Locator="Match(name)" xdt:Transform="InsertIfMissing" lockItem="true" />
       </modules>
     </system.webServer>
   </location>
+  <location path="%XDT_SITENAME%" xdt:Locator="Match(path)" xdt:Transform="Remove" />
```

Comment:
why remove this?

Reply:
I tested this with private site extension and it looks like private site extension still adds location for function app (using xdt from "program files").